<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Measurement</class>
 <widget class="QWidget" name="Measurement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement</string>
  </property>
  <widget class="QLabel" name="label_reptitions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of repetitions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_repetitions">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sampleTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample time (seconds)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sampleTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_purgeTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Purge time (seconds)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_purgeTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_advanced">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
